### PR TITLE
Fix admin UI issues

### LINKED ISSUE #1496: Admin UI: Login button shows inactive
Login button shows inactive if there is a password manager in use and the field has been already filled.

image

### LINKED ISSUE #1497: Admin UI: Tour banner is wrong
Replace this when the new version is available

image

### LINKED ISSUE #1500: Admin UI: fix login and logout for mobile
Make the content expand a little to use more space and remove the extra margin/padding from the top

image from ios

### PR DESCRIPTION
**Pull request by maximlion**
- Admin: Remove disabled button feature from login
Fixes #1496 

-  Admin: Replace old welcome banner image
Self-explanatory 
Fixes #1497 
screenshot from 2018-09-14 22-13-08

- Admin: Fix authentication pages for mobile devices
I had to use "unconventional" methods for this to make all devices work as they should and they all appear to be working great.
Fixes #1500 
screenshot from 2018-09-18 00-01-15

### CODE REVIEW

File: shuup/admin/static_src/base/scss/shuup/auth.scss [hunk 1]
```diff
@@ -1,11 +1,18 @@
+.auth {
+    @media (max-width: 767.98px) {
```
Comment (Pikkupomo) [inline shuup/admin/static_src/base/scss/shuup/auth.scss]: Would it be possible to use breakpoints on these three `@media` definitions?

File: shuup/admin/templates/shuup/admin/auth/login.jinja [hunk 2]
```diff
@@ -40,10 +40,17 @@
 {% endblock %}
 {% block extra_js %}
 <script>
+
 $(function () {
     var fields = $("input");
 
     fields.each(function() {
+        if ($(this).val().length > 0) {
```
Comment (Pikkupomo) [inline shuup/admin/templates/shuup/admin/auth/login.jinja]: Does this work now on cases where you have automatic field filler (for example password manager)?

File: shuup/admin/templates/shuup/admin/auth/login.jinja [hunk 3]
```diff
@@ -40,10 +40,17 @@
...
+                $('.btn-primary').removeClass('disabled');
+        } else {
+            $('.btn-primary').addClass('disabled');
+        } // DRY Perhaps this could be turned into a function?
```
Comment (tulimaki) [inline shuup/admin/templates/shuup/admin/auth/login.jinja]: Yes looks like duplicate code and yes you should likely clean this up a little bit. Also please do not introduce any new TODOs in the small fixing tasks. Point of these small tasks is to actually deal with these kind of small TODOs we have left behind.

File: shuup/admin/templates/shuup/admin/auth/login.jinja [hunk 4]
```diff
@@ -40,10 +40,17 @@
 {% endblock %}
 {% block extra_js %}
 <script>
+
```
Comment (tulimaki) [inline shuup/admin/templates/shuup/admin/auth/login.jinja]: No need to add this extra line.

File: shuup/admin/static_src/base/scss/shuup/auth.scss (hunk 1 above)
Comment (tulimaki) [reply shuup/admin/static_src/base/scss/shuup/auth.scss]: Applies to the other 2 places in this file too.

File: shuup/admin/static_src/base/scss/shuup/auth.scss [hunk 5]
```diff
@@ -15,6 +32,19 @@
...
     }
+    @include media-breakpoint-down(sm) {
+        margin-bottom: 15px;
+        margin-top: 5px;
+    }
+    @include media-breakpoint-up(sm) {
+        &.ipad {
+            padding-top: 20px;
+        }
+        &.android {
```
Comment (tulimaki) [inline shuup/admin/static_src/base/scss/shuup/auth.scss]: We really can't make the admin login and logout work without checking which device is in use?

Comment (maximlion) [reply shuup/admin/static_src/base/scss/shuup/auth.scss]: It's arguably not necessary. If you remove those the menu wont be in the middle on all devices, it'll be more to the top on some and more to the bottom on other devices. I'm gonna finish the retractable menu issue soon so after that I'm gonna search for better solutions to this.

Comment (chessbr) [reply shuup/admin/static_src/base/scss/shuup/auth.scss]: I guess the real check is the screen height here.. if there is "almost" no space in vertical, remove margin/padding from the top

File: shuup/admin/templates/shuup/admin/auth/logout.jinja [hunk 6]
```diff
@@ -15,4 +15,18 @@
...
+            auth_div.classList.add("logout-mobile");
+        }
+        if(/iPad|Tablet|Nexus/i.test(navigator.userAgent) ) {
+            auth_div.classList.add("logout-tablet");
+        }
+    </script>
+    {% endblock %}
+
 {% endblock %}
+
```
Comment (chessbr) [inline shuup/admin/templates/shuup/admin/auth/logout.jinja]: you don't need this extra line

File: shuup/admin/templates/shuup/admin/auth/logout.jinja [hunk 7]
```diff
@@ -15,4 +15,18 @@
             </div>
         </div>
     {% endcall %}
+    {% block extrajs %}
```
Comment (chessbr) [inline shuup/admin/templates/shuup/admin/auth/logout.jinja]: as Panu said before, I guess this is not necessary

File: shuup/admin/templates/shuup/admin/auth/login.jinja [hunk 8]
```diff
@@ -41,18 +41,12 @@
...
-        $(this).on("change", function() {
-            if ($(this).val().length > 0) {
-                $('.btn-primary').removeClass('disabled');
-            } else {
-                $('.btn-primary').addClass('disabled');
-            }
-        })
-    });
-
+    if(/iPad/i.test(navigator.userAgent) ) {
```
Comment (chessbr) [inline shuup/admin/templates/shuup/admin/auth/login.jinja]: IMO, not necessary

Comment (tulimaki) [reply shuup/admin/templates/shuup/admin/auth/login.jinja]: Let's not do these. But please check what is the real issue here. If it is impossible to align the logo on all devices then maybe hide it in mobile.

Comment (chessbr) [reply shuup/admin/templates/shuup/admin/auth/login.jinja]: the issue is not the logo, but the entire form, there is super extra space on the top on mobile devices so users need to scroll to the it

File: shuup/admin/static_src/base/scss/shuup/auth.scss [hunk 9]
```diff
@@ -1,11 +1,20 @@
...
+    justify-content: center;
+}
 
+.auth-block {
 
     &-content {
         padding: 30px;
         position: relative;
+
+        @include media-breakpoint-down(sm) {
```
Comment (chessbr) [inline shuup/admin/static_src/base/scss/shuup/auth.scss]: this is not right. this should be here:

```py
.auth-block-content {
    @include media-breakpoint-down(sm) {
        padding: 30px 0;
    }
}
```

Comment (maximlion) [reply shuup/admin/static_src/base/scss/shuup/auth.scss]: Updated

**Pull request merged**